source/json/responses/finish_check_callback_document_verification.json

```diff
@@ -24,33 +24,41 @@
                     "personal_details": {
                         "name": {
                             "family_name": "Person",
-                            "given_names": ["Example"]
+                            "given_names": [
+                                "Example"
+                            ]
                         },
                         "dob": "1985-10-26",
                         "nationality": "GBR"
                     }
                 },
                 "verification_result": {
                     "all_passed": true,
-                    "image_checks": [{
-                        "category": "Image Quality",
-                        "type": "Image resolution",
-                        "result": "PASS"
-                    }, {
-                        "category": "Image Quality",
-                        "type": "Image sharpness",
-                        "result": "PASS"
-                    }],
+                    "image_checks": [
+                        {
+                            "category": "Image Quality",
+                            "type": "Image resolution",
+                            "result": "PASS"
+                        },
+                        {
+                            "category": "Image Quality",
+                            "type": "Image sharpness",
+                            "result": "PASS"
+                        }
+                    ],
                     "image_checks_passed": true,
-                    "forgery_checks": [{
-                        "category": "Document Fonts",
-                        "type": "Title uses correct font",
-                        "result": "PASS"
-                    }, {
-                        "category": "Checksum",
-                        "type": "MRZ has a valid checksum",
-                        "result": "PASS"
-                    }],
+                    "forgery_checks": [
+                        {
+                            "category": "Document Fonts",
+                            "type": "Title uses correct font",
+                            "result": "PASS"
+                        },
+                        {
+                            "category": "Checksum",
+                            "type": "MRZ has a valid checksum",
+                            "result": "PASS"
+                        }
+                    ],
```

Comment:
Nit: Would be preferable to avoid making formatting changes in commits which make other changes. It makes the diff a little noisy for the reviewer